.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: General Support - AWS Forums
-    url: https://forums.aws.amazon.com
-    about: Please ask and answer questions here.
+    - name: General Support - AWS re:Post
```

Comment:
nit: extra spaces here

Reply:
Not going to change this. 4 space indent vs 2 space indent doesn't matter.